core/src/Http/HttpClientFactory.cs

```diff
@@ -52,28 +51,12 @@ private static HttpClient CreateHttpClient()
             httpClient.DefaultRequestHeaders.Accept.Clear();
             httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
 
-            return httpClient;
+            _httpClient = httpClient;
         }
 
-        public HttpClient GetHttpClient()
-        {
-            return GetHttpClientStatic();
-        }
-
-        public static HttpClient GetHttpClientStatic()
+        public HttpClient GetHttpClient()
```

Comment:
>GetHttpClient() [](start = 26, length = 15)

property instead?

Reply:
approving PR, however IMO it would be cleaner if this was changed to a property.

---
In reply to: [226472521](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1334#discussion_r226472521) [](ancestors = 226472521)